sdk/attestation/attestation/README.md

```diff
@@ -0,0 +1,27 @@
+## Azure AttestationClient SDK for JavaScript
+
+This package contains an isomorphic SDK for AttestationClient.
+
+### Currently supported environments
+
+- Node.js version 8.x.x or higher
+- Browser JavaScript
+
+### How to Install
+
+```bash
+npm install @azure/attestation
+```
+
+### How to use
+
```

Comment:
can we at least file an issue for filling out the README? I find it's easy to forget later

Reply:
https://github.com/Azure/autorest.typescript/issues/806